client/settings/view.js

```diff
@@ -6,15 +6,15 @@ import React, { PropTypes } from 'react';
 /**
  * Internal dependencies
  */
-import WCCSettingsForm from './wcc-settings-form';
```

Comment:
Done in https://github.com/Automattic/woocommerce-services/pull/1051/commits/75d60873b8b89176e6b58157b63f315678670eea